core/src/main/java/org/apache/iceberg/rest/RESTTableOperations.java

```diff
@@ -93,6 +110,19 @@ enum UpdateType {
       this.current = current;
     }
     this.endpoints = endpoints;
+    this.encryptionManager =
+        EncryptionUtil.createEncryptionManager("keyA", 16, keyManagementClient);
```

Comment:
Temporary. I think we need a REST endpoint to retrieve the ID of a table, if it exists.